core/shared/src/main/scala/matryoshka/implicits/package.scala

```diff
@@ -23,15 +23,28 @@ package object implicits
     with Merge.ToMergeOps
     with Recursive.ToRecursiveOps {
 
+  implicit def toRecursive[T, F[_]](implicit T: Birecursive.Aux[T, F])
+      : Recursive.Aux[T, F] =
+    T.rec
+
+  implicit def toCorecursive[T, F[_]](implicit T: Birecursive.Aux[T, F])
+      : Corecursive.Aux[T, F] =
+    T.corec
```

Comment:
And these implicits expose the ones defined in the type class, so now everything can resolve Rec/Corec operations given a Birec constraint.